build-dev.sh

```diff
@@ -1,9 +1,24 @@
-#Temporary steps until we can fix the caching issue in build context #FIXME
+#Make sure the repos mentioned in comments are cloned in the parent directory of this repo
 
-cd ../audit-protocol-private/ && ./build-docker.sh;
-cd ../fpmm-pooler-internal/ && ./build-docker.sh;
+#cd ../ && git clone https://github.com/PowerLoom/audit-protocol.git;
+cd ../audit-protocol/ && ./build-docker.sh;
+
+#cd ../ && git clone https://github.com/PowerLoom/pooler.git;
+cd ../pooler/ && ./build-docker.sh;
+
+#cd ../ && git clone https://github.com/PowerLoom/pooler-frontend.git;
 cd ../pooler-frontend/ && ./build-docker.sh;
+
+#cd ../ && git clone https://github.com/PowerLoom/offchain-consensus.git;
+cd ../offchain-consensus/ && ./build-docker.sh;
+
+#cd ../ && git clone https://github.com/PowerLoom/ap-consensus-dashboard.git;
```

Comment:
I think we'll have to make `ab-consensus-dashboard` public as well.

Reply:
Yup, that's the idea. Was waiting for the PRs to be merged before doing that.